content/Proposals/Accepted/202106-handbook.md

```diff
@@ -0,0 +1,133 @@
+
+##### Despite strong incentives, writing documentation was proven to be one the most unwanted task among engineers
+
+Demotivation is because our (Google Docs based) process tends to create the following obstacles:
+
+* There are too many side decisions to make, e.g. where to put this documentation, what format to use, how long, how to be on-topic, are we sure this information is not recorded somewhere else? Every, even small decision takes our energy and have the risk of procrastination.
+* There is no review process, so it's hard to maintain a high quality of those documents.
+* Created documentation is tough to consume and discover.
+* Because docs are hard to discover, the documentation tends to be often duplicated, has gaps, or is obsolete.
+* Documents used to be private, which brings extra demotivation. Some of the information is useful for the public audience. Some of this could be useful for external contributors. It's hard to reuse such private docs without recreating them.
+
+All of those make people decide NOT to write documentation but rather schedule another meeting and repeat the same information repeatedly.
+
+On a similar side, anyone looking for information about our teams' work, proposals or project is demotivated to look, find and read our documentation because it's not consistent, not in a single place, hard to discover or not completed.
+
+### Pitfalls of the current solution
+
+* It mainly exists in Google Docs, which has the following issues:
+  * Not everything is in our Team drive.
+  * Even if it's not organized well (or only clear for one person), it's not easily discoverable.
+* Existing Google doc-based documents are hard to consume. The formatting is widely different. Naming is inconsistent.
+* Document creation is rarely actionable. There is no review process, so the effort of creating a relevant document might be wasted, as the document is lost. This also leads to docs being in the half-completed state, demotivating readers to look at it.
+* It's hard to track previous discussions around docs, who approved them (e.g. proposals).
+* It's not public, and it's hard to share best practices with other external and internal teams.
+
+## Goals
+
+Goals and use cases for the solution as proposed in [How](#how):
+
+* Single source of truth for Observability Group Team docs.
+* Have a consistent documentation format that is readable and understandable.
+* Searchable and easily discoverable.
+* Process of adding documents should be easy and consistent.
+* Automation and normal review process should be in place to ensure high quality (e.g. link checking).
+* Allow public collaboration on processes and other docs.
+
+### Audience
+
+The currently planned audience for proposed documentation content is following (in importance order):
+
+1. Observability Group Team Members (Monitoring Scrum Teams, optionally Logging and Tracing if they choose to follow our process).
+* Also primary writer.
+1. External Teams and Red Hat.
```

Comment:
```suggestion
2. External Teams at Red Hat.
```
I am not sure if you really meant `and Red Hat`, but I would expect internal customers to come second here.